solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,190 @@
+        costumers_data (dictionary): Bank costumer data.
+
+    Returns:
+        boolean: True for success withdrawal action, False otherwise.
+    """
+    amount_to_withdrawal = float(input("Amount to withdrawal: "))
+    balance_now = float(costumers_data[costumer_id]['balance'])
+    if 0 < amount_to_withdrawal <= balance_now:
+        new_balance = balance_now - amount_to_withdrawal
+        costumers_data[costumer_id]['balance'] = new_balance
+        return True
+
+    return False
+
+
+def cash_deposit(costumer_id, costumers_data):
+    """Deposit money to the bank.
+
+    Args:
+        costumer_id (string): Costumer id.
+        costumers_data (dictionary): Bank costumer data.
+
+    Returns:
+        boolean: True for success deposit action, False otherwise.
+    """
+    amount_to_deposit = float(input("Amount to deposit: "))
+    if amount_to_deposit > 0:
+        balance_now = float(costumers_data[costumer_id]['balance'])
+        new_balance = balance_now + amount_to_deposit
+        costumers_data[costumer_id]['balance'] = new_balance
+        return True
+
+    return False
+
+
+def change_password(costumer_id, costumers_data, new_password):
+    """Changing costumer's ATM password.
+
+    Args:
+        costumer_id (string): Costumer id.
+        costumers_data (dictionary): Bank costumer data.
+        new_password (string): Password the costumer wants to change to.
+
+    Returns:
+        boolean: True for success changing the password, False otherwise.
+    """
+    if len(new_password) == 4 and new_password.isdigit():
+        costumers_data[costumer_id]['password'] = new_password
+        password_now = new_password
+        return password_now
+
+    return False
+
+
+def create_dictionary(path_of_file):
+    """Create dictionary of bank customer data.
+
+    Args:
+        path_of_file (string): Path of file that contains all bank costumer
+            data.
+
+    Returns:
+        dictionary: A nested dictionary of bank costumer data, or print a
+            message and return a None dict if the file path is not correct.
+    """
+    if os.path.isfile(path_of_file):
+        with open(path_of_file, 'r') as file_j:
+            data = json.load(file_j)
+            costumers_data = data
+            return costumers_data
+
+    print("File not found")
+    costumers_data = None
+    return costumers_data
+
+
+def check_costumer_choice(costumer_choice, costumers_data, costumer_id,
+                          costumer_password):
+    """Check costumer choice and act accordingly.
+
+    Args:
+        costumer_choice (integer): Costumer choice from 1/2/3/4/5 options,
+            references to the action he wants to do in ATM.
+        costumers_data (dictionary): Bank costumer data.
+        costumer_id (string): Costumer id.
+        costumer_password (string): Current costumer password.
+
+    Returns:
+        string: Updated password if the choice is 4, or updated id and
+            password if the choice is 5.
```

Comment:
This is not correct.
Your function can return either None, string or tuple.
Your function should return only one thing